modules/3d-tiles/src/lib/parsers/parse-3d-tile-point-cloud.js

```diff
@@ -204,9 +204,16 @@ async function parseDraco(tile, featureTable, batchTable, options, context) {
 // eslint-disable-next-line complexity, max-statements
 export async function loadDraco(tile, dracoData, options, context) {
   const {parse} = context;
+  const dracoOptions = {...options};
+
```

Comment:
Since we are creating a dracoOptions object here, also populate it with `extraAttributes` here?

Reply:
done